src/main/kotlin/com/cognifide/gradle/aem/instance/InstancePlugin.kt

```diff
@@ -41,6 +40,7 @@ class InstancePlugin : AemPlugin() {
             register<InstanceCreate>(InstanceCreate.NAME) {
                 dependsOn(Resolve.NAME)
                 mustRunAfter(InstanceDestroy.NAME)
+                finalizedBy(InstanceUp.NAME)
```

Comment:
finalized by works in a same way as finally block in java so that this is wrong.. if "create" will fail running "up" has no sense

Reply:
check my solution to this, it looks little clunky but it does the job.